src/libraries/Microsoft.PowerFx.Core/Logging/StructuralPrint.cs

```diff
@@ -81,12 +81,18 @@ public override LazyList<string> Visit(FirstNameNode node, Precedence parentPrec
                 return LazyList<string>.Of(sanitizedName);
             }
 
-            var info = _binding?.GetInfo(node);
+            var info = _binding?.GetInfo(node);
+
+            if (IsExpandedType(_binding, node))
+            {
+                return LazyList<string>.Of($"#$fne$#");

```

Comment:
>$"#$fne$#" [](http://example.com/codeflow?start=43&length=10)

Maybe add a comment on the meaning of this string

Reply:
Comment added.